project-koku/koku
# Parquet processing ENV

Adding `ENABLE_PARQUET_PROCESSING` environment variable to signal when we want to do csv->parquet conversion and in the future, summary processing.

**Testing**
1. Enable S3 Archiving env and ensure parquet conversion and table creation is not done
```
(koku2) ➜  koku2 git:(parquet_summary_trigger) echo $ENABLE_PARQUET_PROCESSING
False
(koku2) ➜  koku2 git:(parquet_summary_trigger) echo $ENABLE_S3_ARCHIVING      
True

(koku2) ➜  koku2 git:(parquet_summary_trigger) ✗ ls testing/parquet_data/koku-bucket/data/    
csv
(koku2) ➜  koku2 git:(parquet_summary_trigger) ✗ 
```

2. Enable Parquet Conversion env and ensure csv and parquet files are uploaded to S3
```
(koku2) ➜  koku2 git:(parquet_summary_trigger) ✗ ls testing/parquet_data/koku-bucket/data 
csv     parquet
(koku2) ➜  koku2 git:(parquet_summary_trigger) ✗ 
```
